### PR TITLE
Fix the labelYOffset documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Review by sebr72 — APPROVED** [review]
Thanks.

File: core/src/main/java/org/mapfish/print/map/style/json/MapfishStyleParserPlugin.java [hunk 1]
```diff
@@ -279,9 +279,9 @@
  *       </ul>
  *   <li><strong>labelRotation</strong> (ECQL) - (Point Placement) the rotation of the label
  *   <li><strong>labelXOffset</strong> (ECQL) - (Point Placement) the amount to offset the label
- *       along the x axis. negative number offset to the left
+ *       along the x axis. positve number offset to the right
```
Comment (jwkaltz) [inline core/src/main/java/org/mapfish/print/map/style/json/MapfishStyleParserPlugin.java:282]: positive

File: core/src/main/java/org/mapfish/print/map/style/json/MapfishStyleParserPlugin.java [hunk 2]
```diff
@@ -279,9 +279,9 @@
...
  *   <li><strong>labelYOffset</strong> (ECQL) - (Point Placement) the amount to offset the label
- *       along the y axis. negative number offset to the top of the printing
+ *       along the y axis. positve number offset to the top of the printing
```
Comment (jwkaltz) [inline core/src/main/java/org/mapfish/print/map/style/json/MapfishStyleParserPlugin.java:284]: positive

File: core/src/main/java/org/mapfish/print/map/style/json/MapfishStyleParserPlugin.java (hunk 1 above)
Comment (sbrunner) [reply core/src/main/java/org/mapfish/print/map/style/json/MapfishStyleParserPlugin.java:282]: Thanks -> https://github.com/mapfish/mapfish-print/pull/3287

**Pull request merged**